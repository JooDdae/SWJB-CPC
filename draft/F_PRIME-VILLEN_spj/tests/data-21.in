881502343268712178976494671877119373515743787735572639029859920997529682734635503695926927749738599187915971901527976871153836327771427199367634999535125519689915758276571928117239418147673465771593259269994194213794571627194975291433223917327134787238938339551166730727735578273385187472377111780598147266380944091631971743718297253569517924474875659826970279718113253
