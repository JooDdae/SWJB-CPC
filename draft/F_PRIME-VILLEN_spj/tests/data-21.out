189
152 674 14 390 0 1 0 4 24 971 91 0 151 0 8 2 4 2 3 0 3 4 14 138 4 2 3 3 4 56 1122 1071 317 4 3 10 878 2 90 3 96 893 57 1 4 4 666 1071 289 0 17 4 291 3 1 22 975 0 0 3 2 1047 1 4 4 4 225 46 2 871 669 3 2 3 0 1 3 3 8 887 0 3 4 3 144 850 1149 0 4 1 2 160 1023 19 2 630 4 0 17 2 1 17 393 1165 82 2 22 88 258 296 4 3 10 0 14 2 1 52 0 4 2 1 4 0 2 643 1 2 24 2 23 2 1183 262 63 1 4 4 2 3 3 4 144 2 534 1091 1 2 4 4 0 0 0 4 1013 1023 386 140 600 38 8 4 0 4 14 4 0 145 4 1 3 2 104 3 0 22 363 93 3 120 1037 904 25 42 0 2 1 3 2 